src/microbe_stage/PatchMap.cs

```diff
@@ -25,15 +25,7 @@ public class PatchMap : ISaveLoadable
     public Dictionary<int, PatchRegion> Regions { get; private set; } = new();
 
     [JsonIgnore]
-    public IEnumerable<KeyValuePair<int, PatchRegion>> NormallyDrawnRegions =>
-        Regions.Where(r => !r.Value.UsesSpecialLinking);
-
-    /// <summary>
-    ///   These regions use special logic on how they are linked and positioned to other regions (and drawn)
-    /// </summary>
-    [JsonIgnore]
-    public IEnumerable<KeyValuePair<int, PatchRegion>> SpeciallyDrawnRegions =>
-        Regions.Where(r => r.Value.UsesSpecialLinking);
+    public Vector2 Center { get; private set; }
```

Comment:
I see that this is basically used only in one place? If so I think this should be auto computed property, and then in the map drawer you should just use a normal variable to compute this value once when needed. I think that'll be easier in the future as I suspect our maps won't be static forever so that just stops a bug where this is out of date.

Reply:
I think I update this every time a new region is added.

Reply:
Actually, i've already changed that...